File: src/main/webapp/js/app/messages/messageService.js
app.service("messageService", function ($http) {

    var _this = this;

    _this.findAll = function (receiverId) {
        var params = {
            receiverId: receiverId
        };
        return $http.get("/messages", {params: params}).then(function (response) {
            return response.data;
        })
    }
});

File: src/main/java/uz/unicon/websocket_demo/messages/MessageService.java
package uz.unicon.websocket_demo.messages;


import uz.unicon.websocket_demo.base.BaseService;

import java.awt.print.Pageable;
import java.security.Principal;

public interface MessageService extends BaseService<Message>{
    Message create(IncomingMessage message, Principal principal);
    Iterable<Message>findAll(Long receiverId, Principal principal);
}
